bundles/de.uka.ipd.sdq.simulation.abstractsimengine/src/de/uka/ipd/sdq/simulation/abstractsimengine/SimpleEventBasedSimEntity.java

```diff
@@ -0,0 +1,61 @@
+package de.uka.ipd.sdq.simulation.abstractsimengine;
+
+import java.util.Optional;
+
+/**
+ * A simplified version of {@link AbstractSimEntityDelegator}.
+ * 
+ * The entity supports re-occuring schedules. After being activated, the entityRouting is executed
+ * at the specified initialDelay. If the entity should be rescheduled, the entityRoutine needs to
+ * return the amount of time to the next occurence.
+ * 
+ * @author Sebastian Krach
+ */
+public abstract class SimpleEventBasedSimEntity implements IEntity {
```

Comment:
do we need an additional test case for this class?

Reply:
As we do not have any tests executed right now (which we should reconsider), i forewent creating an additional test case.